Source/Core/Core/IOS/FS/FileSystem.cpp

```diff
@@ -0,0 +1,22 @@
+// Copyright 2018 Dolphin Emulator Project
+// Licensed under GPLv2+
+// Refer to the license.txt file included.
+
+#include "Core/IOS/FS/FileSystem.h"
+
+#include "Core/IOS/FS/HostBackend/FS.h"
+
+namespace IOS::HLE::FS
+{
+std::unique_ptr<FileSystem> MakeFileSystem(IOSC& iosc)
```

Comment:
It doesn't seem like `iosc` is used. Is this intended to be used later?

Reply:
Yeah, this commit originally came from another branch and I forgot to remove the (currently) unused argument :x 

I have removed it now.